### ISSUE TITLE
RFE: indicate that "Undo close group" is about to expire

### ISSUE THREAD

**Issue by leggewie** [Quicksaver/Tab-Groups#310]
When closing a group with the X button there is an option to undo this for a few seconds.  That's very hand and important.  Sometimes I close many groups at once, but especially when I am in single-view sometimes previously closed groups' "Undo close group" option expires and the position of the tab group changes.  When I click the X for another group at the wrong moment, it might be closed inadvertently.  Not a really big deal since I can restore it easily, but still an avoidable nuisance if the tab group's background turned to a different colour 1 second before the option expires or something like that.

**Comment by Quicksaver** [comment]
True. In fact, it's supposed to not expire (disappear) if you move the mouse over it; I'll have to confirm that is actually happening. But in single view it's probably better if no groups expire at all if you're interacting with the top group selectors.

**Comment by iShyboy** [comment]
In single view the bar should only be updated (expires) after exiting the tab view. That's more handy.
Other thing, this "undo close group" could be better.
Something like this:
![1](http://i.imgur.com/BDcvYvT.png)

**Comment by claell** [comment]
+1
